draft-ietf-oauth-transaction-tokens.md

```diff
@@ -306,34 +312,47 @@ In the JWT Header:
 ~~~
 {: #figtxtokenheader title="Example: Txn-Token Header"}
 
-## JWT Body {#txn-token-body}
+## JWT Body Claims {#txn-token-claims}
+
+The transaction token body follows the JWT format and includes existing
+JWT claims as well as defines new claims. These claims are described below:
+
+`iss`:
+: OPTIONAL The `iss` claim as defined in {{RFC7519}} is not required as Txn-Tokens are bound to a single trust domain as defined by the `aud` claim and often the signing keys are known. The `iss` claim MAY be used in cases where the signing keys are not predetermined or it is desired that the Txn-Token Service signs with unique keys.
+
+`iat`:
+: REQUIRED The issued at time of the Txn-Token as defined in {{RFC7519}
+
+`aud`:
+: REQUIRED This claim, defined in {{RFC7519}}, contains the trust domain in which the Txn-Token is valid
+
+`exp`:
+: REQUIRED Expiry time of the Txn-Token as defined in {{RFC7519}}
 
-### Required Claims {#txn-token-claims}
-The JWT body MUST have the following claims:
+`txn`:
+: REQUIRED A unique transaction identifier as defined in Section 2.2 of {{RFC8417}}. When used in the transaction token, it identifies the entire call chain.
 
-* An `iss` claim, whose value is a URN {{RFC8141}} that uniquely identifies the workload or the Txn-Token Service that created the Txn-Token.
-* An `iat` claim, whose value is the time at which the Txn-Token was created.
-* An `aud` claim, whose value is a URN {{RFC8141}} that uniquely identifies the audience of the Txn-Token. This MUST identify the trust domain in which the Txn-Token is used.
-* An `exp` claim, whose value is the time at which the Txn-Token expires.
-* A `txn` claim, whose value is the unique transaction identifier as defined in Section 2.2 of {{RFC8417}}. When used in the transaction token, it identifies the entire call chain.
-* A `sub_id` claim, whose value is the unique identifier of the user or workload on whose behalf the call chain is being executed. The format of this claim MAY be a Subject Identifier as specified in {{SubjectIdentifiers}}.
-* An `azd` claim, whose value is a JSON object that contains values that remain constant in the call chain.
+`sub`:
+: REQUIRED A unique identifier for the subject as defined by the `aud` trust domain. Unlike OpenID Connect, the `sub` claim is NOT associated with the `iss` claim.
 
-### Optional Claims
-The JWT body MAY have the following claims:
+`purp`:
+: REQUIRED A string defining the purpose or intent of this transaction.
 
-#### Requester Context {#requester-context}
-The Txn-Token MAY contain an `req_ctx` claim, whose value is a JSON object the describes the requester context of the transaction. This MAY include the IP address information of the originating user, as well as information about the computational entity that requested the Txn-Token.
+`azd`:
+: OPTIONAL A JSON object that conatains values that remain immutable throughout the call chain.
+
+`req_ctx`:
```

Comment:
For compactness, should we change this name to "rctx" (since we are not using that in the Txn-Token request anymore) or something shorter?

Reply:
I'm ok with that.